lib/interface.js

```diff
@@ -81,31 +82,18 @@ module.exports = function(Target) {
     );
   };
 
+  Target.prototype.asCallback = function(cb) {
+    const promise = this.then();
+    callbackify(() => promise)(cb);
+    return promise;
+  };
+
   // Creates a method which "coerces" to a promise, by calling a
   // "then" method on the current `Target`
-  each(
-    [
-      'bind',
-      'catch',
-      'finally',
-      'asCallback',
-      'spread',
-      'map',
-      'reduce',
-      'thenReturn',
-      'return',
-      'yield',
-      'ensure',
-      'reflect',
-      'get',
-      'mapSeries',
-      'delay',
-    ],
-    function(method) {
-      Target.prototype[method] = function() {
-        const promise = this.then();
-        return promise[method].apply(promise, arguments);
-      };
-    }
-  );
+  availablePromiseMethods().forEach(function(method) {
```

Comment:
The list produced by `availablePromiseMethods()` is significantly shorter than the list that it's replacing.  Was this intentional?  (Granted: I'm assuming that many of the methods in the original list were Bluebird-specific)

Reply:
Yes, this methods are from bluebird